### PR TITLE
Bump Istio to 1.2

### PR DESCRIPTION
**Pull request by scothis**

### CODE REVIEW

**Comment by trisberg** [comment]
Testing this change I get

```
  Warning  Failed     1s (x3 over 45s)   kubelet, gke-riff-next-default-pool-4b51d237-s416  Failed to pull image "docker.io/istio/servicegraph:1.2.5": rpc error: code = Unknown desc = Error response from daemon: manifest for istio/servicegraph:1.2.5 not found
  Warning  Failed     1s (x3 over 45s)   kubelet, gke-riff-next-default-pool-4b51d237-s416  Error: ErrImagePull
```

**Comment by trisberg** [comment]
looks like 1.2.5 pulls in something called `servicegraph` that fails to start due to the above error

File: charts/istio/values.yaml [hunk 1]
```diff
@@ -1,9 +1,10 @@
-# from https://github.com/knative/serving/blob/master/third_party/istio-1.1.7/values-lean.yaml
+# derived from https://github.com/knative/serving/blob/master/third_party/istio-1.2.4/values-lean.yaml
```
[the remark this answers is not in the document]
Comment (trisberg) [reply charts/istio/values.yaml]: > This is using `1.2.4` while we use `1.2.5` in the Makefile, a bit inconsistent but probably works fine

hmm, I guess it didn't work so fine, at least not for my install

**Comment by scothis** [comment]
ugh, CI for PRs only tests the core runtime. That's not helpful...

**Comment by trisberg** [comment]
Strange, I'm not able to reproduce this, additional attempts only create these pods:

```
istio-system   cluster-local-gateway-68c84849c5-gvhss                1/1     Running   0          2m8s
istio-system   istio-ingressgateway-7984b6dd9b-xbh9t                 2/2     Running   0          2m8s
istio-system   istio-pilot-649bc774cf-vdf5l                          1/1     Running   0          2m8s
```

No idea where that `pod/servicegraph-66bc9c44f4-scq4s` came from, it was definitely created in the `istio-system` namespace on a new cluster, maybe my local build had some old cruft left over

**Comment by trisberg** [comment]
Based on further testing it seems to work as expected, so we are good to go.

File: charts/istio/values.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (scothis) [reply charts/istio/values.yaml]: 1.2.4 is what's in the the Knative Serving repo vs 1.2.5 is the latest patch release

**Comment by scothis** [comment]
same AZP bug strikes again

**Pull request merged**